modules/ossm-upgrading-to-21.adoc

```diff
@@ -57,7 +57,7 @@ For example:
 ----
 spec:
   policy:
-    type: Istiod
+    type: Istoid
```

Comment:
Should this remain as `Istiod`? I am unsure but am mentioning this because `Istiod` is used elsewhere in this doc.